include/SFML/OpenGL.hpp

```diff
@@ -33,7 +33,7 @@
 
 
 ////////////////////////////////////////////////////////////
-/// This file just includes the OpenGL (GL and GLU) headers,
+/// This file just includes the OpenGL headers,
```

Comment:
Is there an issue to use glu in the user code? If not, I'd still include them for convenience.